apps/managedidentity/docs/RUN_IMDS.md

```diff
@@ -0,0 +1,162 @@
+# Running the IMDS Source
+
+This file will talk you through the steps required to run the IMDS Managed Identity Source locally
+
+## Files to Modify
+
+Firstly, to run IMDS locally, we need to change some files.
+
+1. Comment out everything in `client_certificate_sample.go` besides `package main`
+2. In `sample_utils.go`, replace the Config struct with this
+
+```
+type Config struct {
+    ClientID      string   `json:"client_id"`
+    Authority     string   `json:"authority"`
+    Scopes        []string `json:"scopes"`
+    Username      string   `json:"username"`
+    Password      string   `json:"password"`
+    RedirectURI   string   `json:"redirect_uri"`
+    CodeChallenge string   `json:"code_challenge"`
+    // CodeChallengeMethod string   `json:"code_challenge_method"`
+    // State               string   `json:"state"`
+    ClientSecret string `json:"client_secret"`
+    // Thumbprint          string   `json:"thumbprint"`
+    // PemData             string   `json:"pem_file"`
+}
```

Comment:
Need to remove step 1 and 2 if we create a new folder for managed identity sample

Reply:
I think your commenting on the outdated commit